File: tests/Generator/resource/csharp/complex/SecurityOAuth.cs
using System.Text.Json.Serialization;

namespace TypeAPI.Model;

public class SecurityOAuth : Security
{
    [JsonPropertyName("authorizationUrl")]
    public string? AuthorizationUrl { get; set; }

    [JsonPropertyName("scopes")]
    public System.Collections.Generic.List<string>? Scopes { get; set; }

    [JsonPropertyName("tokenUrl")]
    public string? TokenUrl { get; set; }

}

File: tests/Generator/resource/swift/complex/SecurityOAuth.swift
class SecurityOAuth: Security {
    var authorizationUrl: String?
    var scopes: Array<String>?
    var tokenUrl: String?

    enum CodingKeys: String, CodingKey {
        case authorizationUrl = "authorizationUrl"
        case scopes = "scopes"
        case tokenUrl = "tokenUrl"
    }
}

